tests/integration/test_application.py

```diff
@@ -257,3 +257,33 @@ def test_global_environment(
     assert variables["project_name"] == "environment-project"
     assert variables["project_dir"] == str(tmp_path)
     assert variables["project_version"] == "1.2.3"
+
+
+def test_build_secrets_destructive(create_app, monkeypatch, tmp_path, capsys):
```

Comment:
For real though these tests are so easy to write now that it's almost not fun anymore hehe
it takes the _sport_ out of the thing, you know?